entries/accordion.xml

```diff
@@ -86,6 +86,21 @@
 				</desc>
 			</type>
 		</option>
+		<option name="classes" type="Object" default='Object' example-value='{ "ui-accordion-header": null }'>
```

Comment:
This isn't a great example value because it looks like it's just removing the value from one key, but it's effectively removing the value from all keys.